util/util.go

```diff
@@ -475,3 +482,58 @@ func (o MachinesByCreationTimestamp) Less(i, j int) bool {
 	}
 	return o[i].CreationTimestamp.Before(&o[j].CreationTimestamp)
 }
+
+// AddWatchOnClusterPaused adds a conditional watch to the controlled given as input
+// that sends watch notifications on any create or delete, and only updates
+// that toggle Cluster.Spec.Cluster.
+func WatchOnClusterPaused(c controller.Controller, mapFunc handler.Mapper) error {
+	return c.Watch(
+		&source.Kind{Type: &clusterv1.Cluster{}},
+		&handler.EnqueueRequestsFromMapFunc{
+			ToRequests: mapFunc,
+		},
+		predicate.Funcs{
```

Comment:
This should probably also define a CreateFunc that returns false if Cluster.Spec.Paused is true

Reply:
I removed it because it's not a huge problem if the objects are reconciled when a cluster is created (e.g. move) or deleted

Reply:
+1 also previously create func was checking only for the annotation not the field